docs/how-to/release_process/canary.rst

```diff
@@ -0,0 +1,107 @@
+Canary Testing for Checkbox Edge Version: In-depth Process
+==========================================================
+
+Introduction
+------------
+
```

Comment:
It would be good to explain what this whole section of the documentation is about: testing Checkbox snaps that land in the Edge channel in the Snapstore when a new version is built (daily if changes are made to the source code).

(I know this is more or less described later in that page, but it is spread over the page, and it is separated from the "Introduction" section by a TOC that points to other documents)

Reply:
@tang-mm Added explanation in the #887. I think it satisfies your comment.